src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -35,16 +38,23 @@ private ShooterSubsystem(){
         topShooter = new CANSparkMax(20, MotorType.kBrushless);
         bottomShooter = new CANSparkMax(21, MotorType.kBrushless);
         rightElevator = new CANSparkMax(25, MotorType.kBrushless);
-        leftElevator = new CANSparkMax(26, MotorType.kBrushless);
-        kicker = new VictorSPX(19);
-        bottomShooter.follow(topShooter);
-        leftElevator.follow(rightElevator);
+        // leftElevator = new CANSparkMax(26, MotorType.kBrushless);
+        // kicker = new VictorSPX(19);
```

Comment:
I think we can go ahead and remove these commented out lines.

Reply:
done